Tests/WebApi.UnitTests/Applications/ActorOperations/Queries/GetActorDetail/GetActorDetailQueryValidatorTest.cs

```diff
@@ -0,0 +1,36 @@
+using AutoMapper;
+using FluentAssertions;
+using MovieStoreWebApi.Applications.ActorOperations.Queries.GetActorDetail;
+using MovieStoreWebApi.DbOperations;
+using Tests.WebApi.UnitTests.TestSetup;
+using Xunit;
+
+namespace Tests.WebApi.UnitTests.Applications.ActorOperations.Queries.GetActorDetail
+{
+    public class GetActorDetailValidatorTest : IClassFixture<CommonTestFixture>
+    {
+        private readonly MovieStoreDbContext _context;
+        private readonly IMapper _mapper;
+
+        public GetActorDetailValidatorTest(CommonTestFixture testFixture)
+        {
+            _context = testFixture.Context;
+            _mapper = testFixture.Mapper;
+        }
+
+        [InlineData(0)]
+        [InlineData(1)]
+
+        [Theory]
+        public void WhenInvalidInputAreGiven_Validator_ShouldBeReturnErrors(int id)
+        {
+            GetActorDetailById query = new GetActorDetailById(null, null);
```

Comment:
Bu test olmamis, test'in amaci direkt hata almaksa direkt exception firlatmakla ayni sey bu satirda yapilanin sonucu.

Command'ine null argumanlar verdigin icin hata donuyor, input invalid oldugu icin degil.
Command'inin calisabilir olmasi lazim, ama InlineData olarak verdigin datalarin databasede olmayacak idler olmasi lazim.

Reply:
query'e null parametreler veriliyor fakat teorik test ediliyor. Teorik test edilse de metod (int id) parametresi alıyor. Bu parametre metodun üstünde InlineData değerlerinden geliyor. GetActorDetailByIdValidator sınıfında InlineData içerisindeki değerler validate edilerek test ediliyor. 

Burada sadece InlineData içerisinde id örnekleri validate'den geçip geçmediği deneniyor.